<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-OE9ALJNP" failures="2" tests="2" name="Test(failed)" time="36.003" errors="0" timestamp="2024-04-07T16:01:59 IST">
  <testcase classname="TestCase.TC_DataDrivenTest_002" name="verify_loginData" time="11.442">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCase.TC_DataDrivenTest_002.verify_loginData(TC_DataDrivenTest_002.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginData -->
  <testcase classname="TestCase.TC_DataDrivenTest_002" name="verify_loginData" time="11.625">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCase.TC_DataDrivenTest_002.verify_loginData(TC_DataDrivenTest_002.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginData -->
</testsuite> <!-- Test(failed) -->
